Update internal-transfers.md (#455)
mattermost/mattermost-handbook: operations/workplace/people/working-at-mattermost/internal-transfers.md

> # Transfers & Promotions

Reviewer: ```suggestion
# Transfers and Promotions
```

> We want staff to feel empowered to do the best work of their careers at Mattermost and enabled to drive the direction of their own growth trajectories at Mattermost. 

Reviewer: ```suggestion
We want staff to feel empowered to do the best work of their careers at Mattermost and enabled to drive the direction of their own growth trajectories at Mattermost.
```

# Transfers & Promotions

We want staff to feel empowered to do the best work of their careers at Mattermost and enabled to drive the direction of their own growth trajectories at Mattermost. 

## What's the difference between a transfer and a promotion?

> * Transfers occur when a team member moves to a new team or function and owns a different set of [AORs](https://handbook.mattermost.com/operations/operations/areas-of-responsibility). For example, a Customer Support Engineer would transfer to a Customer Success Engineer role, or vice versa.

Reviewer: ```suggestion
* Transfers occur when a team member moves to a new team or function and owns a different set of [AORs](https://handbook.mattermost.com/operations/operations/areas-of-responsibility). For example, a Customer Support Engineer transferring to a Product Manager role.
```

Reviewer: I changed the transfer example so it's clear that transfers aren't only within a specific team or area of expertise. LMK if that's incorrect.

Author: Makes total sense, ty!

> * Promotions occur when a team member increases in level within the same team or function. For example, an SDE I being promoted to SDE II.

Reviewer: ```suggestion
* Promotions occur when a team member increases in level within the same team or function. For example, an SDE I being promoted to SDE II.
```

> ### Process for transfers

Reviewer: ```suggestion
### Process for Transfers
```

> Internal transfers across teams have an informal and a formal component. Informally, a team member is encouraged to be curious and seek to learn about other roles, teams, projects, and Hiring Managers. Managers or MLT may also initiate a transfer conversation as part of general org and team planing.

Reviewer: ```suggestion
Internal transfers have an informal and a formal component. 

Informally, a team member is encouraged to be curious and seek to learn about other roles, teams, projects, and Hiring Managers. Managers or MLT may also initiate a transfer conversation as part of general organization and team planing.
```

> When a team member and Hiring Manager agree that a potential transfer may be a good fit, the Hiring Manager should discuss a potential transfer with the team member's current Manager as well as HR to kick off the formal transfer process.

Reviewer: ```suggestion
When a team member and Hiring Manager agree that a potential transfer may be a good fit, the Hiring Manager should discuss a potential transfer with the team member's current Manager as well as HR to kick off the formal transfer process.
```

- Allow HR to confirm potential compensation impact, especially if the transition results in a decrease in compensation.
- Create a transition plan that balances the needs above with an aligned timeline.

> ### Process for promotions

Reviewer: ```suggestion
### Process for Promotions
```

**Cadence**

> Level changes are reviewed and discussed during every [performance review cycle](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50?q=levels). As such, promotions typically happen once a year during the [Winter Performance Review cycle](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50/performance-review-timeline-winter-2019). Managers and staff are also encouraged to discuss growth and development opportunities throughout the year. 

Reviewer: ```suggestion
Level changes are reviewed and discussed during every [performance review cycle](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50?q=levels). As such, promotions typically happen once a year during the [Winter Performance Review cycle](https://handbook.mattermost.com/operations/workplace/people/performance-reviews-50/performance-review-timeline-winter-2019). Managers and staff are also encouraged to discuss growth and development opportunities throughout the year.
```

**Off-Cycle Cadence**

> Off-cycle promotions may also occur based on business need. This is typically determined if a [new job need exists](https://handbook.mattermost.com/contributors/join-us/staff-recruiting#how-to-open-a-new-role). Staff members can express interest in being considered for an [open role](https://mattermost.com/careers/) by reaching out to the Hiring Manager. Similar to transfers, depending on the new role, there may be a need to assess skillset. This will be determined and communicated on a case-by-case basis, and will typically involve components of the internal interview process. 

Reviewer: ```suggestion
Off-cycle promotions may also occur based on business need. This is typically determined if a [new job need exists](https://handbook.mattermost.com/contributors/join-us/staff-recruiting#how-to-open-a-new-role). Staff members can express interest in being considered for an [open role](https://mattermost.com/careers/) by reaching out to the Hiring Manager. Similar to transfers, depending on the new role, there may be a need to assess skillset. This will be determined and communicated on a case-by-case basis, and will typically involve components of the internal interview process.
```

**Manager and Above Promotions**

> Managers are responsible for ensuring alignment across the org and within their own team. Managers play a significant role in their teams and direct report's careers at Mattermost. It's important that every newly hired or promoted manager fully understands the role and responsibitlies of Managers at Mattermost, and that they are set up for success to meet or exceed future role expectations. We assess every management level based on the [Mattermost Leadership Principles](https://handbook.mattermost.com/company/about-mattermost#leadership-principles).  

Reviewer: ```suggestion
Managers are responsible for ensuring alignment across the org and within their own team. Managers play a significant role in their teams and direct report's careers at Mattermost. It's important that every newly hired or promoted manager fully understands the role and responsibilities of Managers at Mattermost, and that they're set up for success to meet or exceed future role expectations. We assess every management level based on the [Mattermost Leadership Principles](https://handbook.mattermost.com/company/about-mattermost#leadership-principles).
```

* Promotions to Manager will typically follow the typical internal interview process.  
* Promotions to Director will typically include interviews with the CEO and CTO. 

### Typical Internal Interview Process

> Hiring Managers and MLT will determine what the skillset assessment process will be for an internal promotion or transfer. The typical process here is intended to outline the various components of an internal transition. This process should also ocurr after the discussion topics listed above have taken place with approval from the Team MLT.

Reviewer: ```suggestion
Hiring Managers and MLT will determine what the skillset assessment process will be for an internal promotion or transfer. The typical process here is intended to outline the various components of an internal transition. This process should also occur after the discussion topics listed above have taken place with approval from the Team MLT.
```
